### ISSUE TITLE
Adding default landing dashboard

### ISSUE THREAD

**Issue by redet-G** [HABTec/public-dashboard#20]
The first page should show one of the dashboards and start visualizing it. We could assume that the first favorited dashboard is meant to be displayed as the home page.

**Comment by redet-G** [comment]
let's work on this @RihanaE. Let me know if this is not clear.

**Comment by RihanaE** [comment]
Is the favorited dashboard the one whose dashboard item is in the saved reports list?

**Comment by redet-G** [comment]
No, each of the dashboards has an attribute named `favorite`.  From the list of dashboards where this attribute is set as true you would display the first dashboard as a default landing page. 

Here is the JSON structure you would be using `https://hmis-staging.moh.gov.et/api/dashboards?fields=name,id,favorite`
```
    {
      "name": "Adwa Health Center 2016",
      "favorite": false,
      "id": "YwhKTtZ547d"
    }

**Comment by redet-G** [comment]
[3b338d3](https://github.com/HABTec/public-dashboard/commit/3b338d3ebea875505fdae2b05be870c617392f90)
